### ISSUE TITLE
[Suggestion] Adding Turntable Recipes with Minetweaker

### ISSUE THREAD

**Issue by abculatter2** [BeetoGuy/BetterWithMods#214]
Please?

**Comment by WesCook** [comment]
How would this work in practice?  Perhaps an array of blocks with different progress levels?

eg. `<minecraft:red_sand>, [<minecraft:glass>, <betterwithmods:poor_sand>];`

Though that doesn't make it easy to edit existing ones, such as clay.  You'd need "progress level" integers for each item.  Might be simpler if it just overwrote it if clay were to be added again.

**Comment by DaedalusGame** [comment]
This would be trivial, turntable recipes are already in the form of block => block + item

**Comment by primetoxinz** [comment]
have been working on adding this, will be in soonish likely only for 1.11.2, unless a backport is wanted enough

**Comment by primetoxinz** [comment]
This has been pushed and will release soon, info on how to use can be found [here](https://github.com/BeetoGuy/BetterWithMods/wiki/MineTweaker)

**Comment by WesCook** [comment]
I'm going to clean up the formatting on the ModTweaker page, if you wouldn't mind.  It's starting to become hard to read.

**Comment by primetoxinz** [comment]
go for it

**Comment by WesCook** [comment]
Great, thanks.  I've just submitted my changes.

https://github.com/BeetoGuy/BetterWithMods/wiki/MineTweaker

I've redesigned the page around the detail/summary tags.  These allow you to click to expand more information, which should keep the page more readable.

It's worth noting the Edge and Internet Explorer don't currently support these tags, so the content will just be expanded by default.  It still looks fine though.  Chrome, Firefox, Safari and Opera support these tags no problem.

I've added parameter information for each of the machine types.  I hope I got it all correct.  I'd suggest reviewing it when you have a moment, just in case I've missed anything.  I was a bit unsure of how the Filtered Hopper operates, for instance.

Anyway, hope that helps.

**Comment by primetoxinz** [comment]
Oh, btw I did put turntable for 1.10.2.  already fixed that on the wiki as I saw you put 1.11.2 only

**Comment by WesCook** [comment]
I'm not sure what's happened, but it looks like the formatting has been broken on the MineTweaker page.  The revision history was also wiped when the page name was changed, so we can't review previous changes.

https://github.com/BeetoGuy/BetterWithMods/wiki/MineTweaker-(MTLib-required)

I'd guess it was user error, but the code looks similar to before.  Maybe a Github change affected how Markdown is processed?

**Comment by KnightMiner** [comment]
There is a chance you accidently changed the page type from markdown to something else, like it has a mediawiki setting.

**Comment by KnightMiner** [comment]
Otherwise, try going to history and replacing the page name in the URL with the old title

**Comment by WesCook** [comment]
Good suggestions, but the page appears to be in Markdown mode.  It actually looks like Markdown won't execute inside a `<details>` tag now.  It worked before, so I'm not sure what's changed.

I also tried reviewing history of the previous URL, but it throws a 404 now.  So unless the repo owner can access deleted pages, that may be lost.

https://github.com/BeetoGuy/BetterWithMods/wiki/MineTweaker/_history

**Comment by WesCook** [comment]
I played with the different parsers, and found that the existing system will continue to work under Textile pages with minor edits to syntax.

I've created a new page just for comparison.  I don't like that it's not using Markdown, but at least it's functional again.

https://github.com/BeetoGuy/BetterWithMods/wiki/MineTweaker-(Texttile)

I probably should have just made a new issue for this by now, but @primetoxinz let me know your thoughts.

I'm still completely bewildered about what changed with the Markdown parser.  Very strange issue.

**Comment by primetoxinz** [comment]
no idea what the issue with Markdown is, github may have changed something that made it not work. anyways might as well just use the one that works, texttile will do

**Comment by WesCook** [comment]
Using multiple cursors (<3 Sublime Text), it's not too hard to change between formats.  Plus it should remain in revision history, so I'm not too worried about keeping it.

Though if we did want to change the title, I was actually going to suggest the page be renamed from MineTweaker to CraftTweaker at some point.  That branding has mostly been changed in the v1.10 world, so I think it would make more sense.

If that sounds like a plan, I'll go ahead and update the page to CraftTweaker branding and use the new Textile format then.  And I'll just delete the second page I made.  Does that work?
